gsy_myco_sdk/matching_algorithms/attributed_matching_algorithm.py

```diff
@@ -50,23 +50,23 @@ def get_matches_recommendations(
                 # Trading partners matching
                 trading_partners_recommendations = (
                     PreferredPartnersMatchingAlgorithm.get_matches_recommendations(
-                        {market_id: data}))
+                        {market_id: {time_slot: data}}))
```

Comment:
Given all the recent rebranding I lost track of this: what is `market_id`? Is it the ID of a market (in the sense of a market object that contains trades, IAAs, etc) or is the ID of a "market" in the sense of area (like "House 1")?

Reply:
It's the area id (the area with no strategy)